gradle.properties

```diff
@@ -1,6 +1,6 @@
-ideaVersion=2020.2
+ideaVersion=LATEST-EAP-SNAPSHOT
```

Comment:
I assume there was a reason for it being 2020.2 right?
I think once the 2020.3 build comes out we can switch to that.

However, this is needed to fix the compile issue with a portion of the code.